### PR TITLE
Create a VPA chart. E2e test on 1.18 and not 1.15.

### PR DESCRIPTION
**Pull request by sudermanjr**
A chart for https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler

**Why This PR?**

Right now the only way to install VPA is with their bash scripts via the hack directory in the VPA repo.

### CODE REVIEW

File: stable/vpa/templates/recommender-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.recommender.securityContext | nindent 12 }}
+          image: "{{ .Values.recommender.image.repository }}:{{ .Values.recommender.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.recommender.image.pullPolicy }}
+          {{- if .Values.recommender.extraArgs }}
+          args:
+          {{- range $key, $value := .Values.recommender.extraArgs }}
+            - --{{ $key }}={{ $value }}
```
Comment (rbren) [inline stable/vpa/templates/recommender-deployment.yaml:41]: maybe `quote` for safety

**Review by rbren — APPROVED** [review]

File: stable/vpa/templates/recommender-deployment.yaml (hunk 1 above)
Comment (sudermanjr) [reply stable/vpa/templates/recommender-deployment.yaml:41]:
```suggestion
            - --{{ $key }}={{ quote $value }}
```

File: stable/vpa/templates/recommender-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.recommender.securityContext | nindent 12 }}
+          image: "{{ .Values.recommender.image.repository }}:{{ .Values.recommender.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.recommender.image.pullPolicy }}
+          {{- if .Values.recommender.extraArgs }}
+          args:
+          {{- range $key, $value := .Values.recommender.extraArgs }}
+            - --{{ $key }}={{ quote $value }}
```
Comment (sudermanjr) [inline stable/vpa/templates/recommender-deployment.yaml]:
```suggestion
            - --{{ $key }}={{ $value | quote }}
```

File: stable/vpa/templates/recommender-deployment.yaml (hunk 1 above)
Comment (sudermanjr) [reply stable/vpa/templates/recommender-deployment.yaml:41]: I tried that and it failed e2e tests, so I dropped it.

**Review by lucasreed — APPROVED** [review]
LGTM, validated by running on local minikube and installing goldilocks works with this vpa chart installed first.

**Pull request merged**